stacker/lookups/output.py

```diff
@@ -0,0 +1,35 @@
+from collections import namedtuple
```

Comment:
worth moving this into a subdirectory called `handlers/` or something like that?